packages/backend/discovery/across/config.jsonc

```diff
@@ -33,14 +33,21 @@
         }
       }
     },
+    "0xB524735356985D2f267FA010D681f061DfF03715": {
+      "ignoreInWatchMode": ["nonce"]
+    },
     "0x592349F7DeDB2b75f9d4F194d4b7C16D82E507Dc": {
-      "ignoreMethods": ["proposals", "getProposal"]
+      "ignoreMethods": ["proposals", "getProposal", "getCurrentTime"],
+      "ignoreInWatchMode": ["numProposals"]
     },
     "0x226726Ac52e6e948D1B7eA9168F9Ff2E27DbcbB5": {
-      "ignoreMethods": ["bondedProposals"]
+      "ignoreMethods": ["bondedProposals", "getCurrentTime"]
     },
     "0x4D9079Bb4165aeb4084c526a32695dCfd2F77381": {
-      "ignoreMethods": ["rootBundles"]
+      "ignoreMethods": ["rootBundles", "getCurrentTime", "numberOfDeposits"]
+    },
+    "0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828": {
+      "ignoreInWatchMode": ["totalSupply"] // Maybe we should also ignore it in discovery?
```

Comment:
Current approach is that we monitor (almost) everything in discovery, and ignore in watchmode things that change too often.

Reply:
If you say so then should I move getCurrentTime and numberOfDeposits from ignoreMethods to ignoreInWatchMode?

Reply:
Yes, that's what we currently do. I like your approach, but right now `ignoreMethods` are used so `discovery` doesn't go too deep.